matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/EventType.kt

```diff
@@ -49,6 +49,7 @@ object EventType {
     const val STATE_ROOM_JOIN_RULES = "m.room.join_rules"
     const val STATE_ROOM_GUEST_ACCESS = "m.room.guest_access"
     const val STATE_ROOM_POWER_LEVELS = "m.room.power_levels"
+    const val STATE_ROOM_BEACON_INFO = "m.beacon_info"
```

Comment:
There is no unstable prefix?

Reply:
(see https://github.com/matrix-org/matrix-spec-proposals/blob/stefan/ephemeral-location-streaming/proposals/3672-ephemeral-location-streaming.md#unstable-prefix)

Reply:
Oh no, of course there is. Thanks for catching.

Reply:
Fixed.